File: projects/m1/003-making-change/solutions/PiervitoRinaldi/making-change.py
# MAKING CHANGE
# Calcola il resto dovuto, da una cassa automatica, con il minor numero di monete possibili.
# toonie= 2$ loonie= 1$ quarter= 25 cent dime= 10 cent nickel= 5 cent penny= 1 cent

cash= int(input)    # Inserisci in centesimi la quantità che stai versando
costo= int(input)   # Inserisci il costo della tua spesa
resto= int(cash-costo)

toonies= resto//200
loonies= (resto%200)//100
quarters= ((resto%200)%100)//25
dimes= (((resto%200)%100)%25)//10
nickels= ((((resto%200)%100)%25)%10)//5
pennies= ((((resto%200)%100)%25)%10)%5

print(toonies,'toonies \n', loonies,'loonies \n', quarters,'quarters \n', dimes,'dimes \n', nickels,'nickels \n', pennies,'pennies')

File: projects/m1/003-making-change/solutions/biagio/making-change.js
const prompt = require("prompt-sync")();

var money = prompt("How many cents do you have? ");

/*
1 penny = 1 cent $
1 nickel = 5 cent $
1 dime = 10 cent $
1 quarter = 25 cent $
1 loonie = 1$
1 toonie = 2$
*/

var toonie = Math.floor(money / 200);
var money = money % 200;
var loonie = Math.floor(money / 100);
var money = money % 100;
var quarter = Math.floor(money / 25);
var money = money % 25;
var dime = Math.floor(money / 10);
var money = money % 10;
var nickel = Math.floor(money / 5);
var money = money % 5;
var penny = Math.floor(money / 1);



console.log(toonie + " toonie, " + loonie + " loonie, " + quarter + " quarter, " + dime + " dime, " + nickel + " nickel, " + penny + " penny.")